core/src/main/resources/reference.conf

```diff
@@ -229,7 +229,7 @@ master {
     }
     cluster {
       roles = ["master"]
-      auto-down-unreachable-after = 15s
+      auto-down-unreachable-after = 90s
```

Comment:
Better not change options in core. For testing we need to override the option value, if it is really necessary.